google/cloud/ndb/key.py

```diff
@@ -739,6 +739,29 @@ def urlsafe(self):
         raw_bytes = self.serialized()
         return base64.urlsafe_b64encode(raw_bytes).strip(b"=")
 
+    def to_legacy_urlsafe(self, location_prefix=None):
```

Comment:
If this method has no use except to add the location prefix, we can make the location prefix non-optional.